.eslintrc.js

```diff
@@ -1,7 +1,10 @@
 module.exports = {
-  "env": {
-    "es6": true,
-    "node": true
+  env: {
+    es6: true,
```

Comment:
this is in the base rule set: https://github.com/digitalbazaar/eslint-config-digitalbazaar/blob/master/index.js#L3